packages/devextreme/js/__internal/ui/chat/messagebox.ts

```diff
@@ -81,10 +112,12 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
       autoResizeEnabled: true,
       valueChangeEvent: 'input',
       maxHeight: '8em',
-      onInput: (): void => {
+      onInput: (e: InputEvent): void => {
         const shouldButtonBeDisabled = !this._isValuableTextEntered();
 
         this._toggleButtonDisableState(shouldButtonBeDisabled);
+
+        this._onInputTriggerTypingEventsHandler(e);
```

Comment:
It's unclear why we use 'onInput' & 'Events' inside this method name. Can we adjust this name?